daemon/daemon.go

```diff
@@ -400,7 +401,11 @@ func (d *Daemon) addRoutes() {
 
 	for _, c := range api {
 		c.d = d
-		d.router.Handle(c.Path, c).Name(c.Path)
+		if c.PathPrefix == "" {
+			d.router.Handle(c.Path, c).Name(c.Path)
+		} else {
+			d.router.PathPrefix(c.PathPrefix).Handler(c)
```

Comment:
can we minimally test this? for example for /v2/debug/pprof/cmdline

Reply:
I'll add some unit test and maybe a spread test too.